lightning/src/events/mod.rs

```diff
@@ -480,12 +480,16 @@ pub enum HTLCHandlingType {
 		channel_id: ChannelId,
 	},
 	/// Scenario where we are unsure of the next node to forward the HTLC to.
+	///
+	/// Deprecated: will only be used in versions before LDK v0.2.0.
```

Comment:
I think you can use the official `deprecated` attribute on this: https://doc.rust-lang.org/reference/attributes/diagnostics.html#r-attributes.diagnostics.deprecated

Reply:
TIL this exists! 

Struggling to `#[allow(deprecated)]` for the `impl_writeable_tlv_based_enum_upgradable` invocation to silence the cargo warning though..